README.md

```diff
@@ -1,2 +1,374 @@
+
+```elixir
+# ❌ Bad
+defmodule Jungsoft.BeautifulOfficeView do
+  @moduledoc false
+
+  def foo do
+
+    1 + 1
+
+  end
+end
+
+
+# ✅ Good
+defmodule Jungsoft.BeautifulOfficeView do
+  @moduledoc false
+
+  def foo do
+    1 + 1
+  end
+end
+```
+
+#### With
+
+The `with do` parameters should not be separated by blank spaces. Except when `else` contains multiple options. In this case should use the same logic as `case` and `cond` normal style (see [here](https://github.com/christopheradams/elixir_style_guide#multiline-case-clauses)).
+
+```elixir
+# ❌ Bad
+with  %StarWars{} <- Jungsoft.function_1(),
+      true <- Jungsoft.function_2()
+do
+
+  ## DO SOMETHING
+
+else
+
+  %StarTrek{} ->
+    {:error, "Wrong choice"}
+
+  false ->
+    ## DO SOMETHING
+
+end
+
+
+# ✅ Good
+with  %StarWars{} <- Jungsoft.function_1(),
+      true <- Jungsoft.function_2()
+do
+  ## DO SOMETHING
+else
+  %StarTrek{} ->
+    {:error, "Wrong choice"}
+
+  false ->
+    ## DO SOMETHING
+end
+```
+
+[Back to top ⬆️](#table-of-contents)
+
+## Dangling Commas
+
+Objects with the possibility to span in multiple lines should contain a comma dangle in the last parameter. See an alias example below.
+
+```elixir
+# ❌ Bad
+alias Jungsoft.{
+  Module1,
+  Module2
+}
+
+
+# ✅ Good
+alias Jungsoft.{
+  Module1,
+  Module2,
+}
+```
+
+**This is also valid for Maps, Lists and Keyword Lists.**
+
+[Back to top ⬆️](#table-of-contents)
+
+# Schemas
+
+The schemas should respect a pattern for organizing the parameters.
+
+Independent of the schema type, the parameters should be grouped by type:
+```elixir
+# ❌ Bad
+schema "foo" do
+  field :name, :string
+
+  field :age, :integer
+
+  has_one :one, Bar
+
+  has_one :two, Module
+  timestamps()
+end
+
+
+# ✅ Good
+schema "foo" do
+  field :name, :string
+  field :age, :integer
+
+  has_one :one, Bar
+  has_one :two, Module
+
+  timestamps()
+end
+```
+
+The other patterns are different based on the type of schema (Ecto or GraphQL). These differences are listed below.
+
+## Ecto Schema
+
+The Ecto schema parameters need to be organized in a order of importance:
+
+| Importance | Field |
+|---|---|
+| 1º | field |
+| 2º | belongs_to |
+| 3º | has_one |
+| 4º | has_many |
+| 5º | timestamps |
+
+In this way, `field` should be inserted before `belongs_to`, and so on.
+
+## GraphQL Schema
+
+The GraphQL schema parameters need to be organized in a order of importance:
+
+| Importance | Field |
+|---|---|
+| 1º | normal fields |
+| 2º | custom fields (with the `resolve`) |
+| 3º | associations |
+
+In this way, `normal fields` should be inserted before `custom fields`, and so on.
+
+Remember, the first schema rule should be respected here as well (see [here](#schemas)). Therefore, `normal fields` should be grouped and separated with a blank space. Same is valid with `custom fields` and `associations`.
+
+[Back to top ⬆️](#table-of-contents)
+
+# Regular Expressions
```

Comment:
I'm not sure this is the best name, because when I read "regular expressions" I immediately think of regex, which is not the case.